vaccine_feed_ingest/runners/me/maine_gov/normalize.py

```diff
@@ -73,7 +80,7 @@ def _get_contacts(site: dict) -> List[schema.Contact]:
 
     website_matches = re.search('href="(http.*)"', scheduling_info_raw)
     if website_matches:
-        website = website_matches.group(1)
+        website = website_matches.group(1).split(" ")[0]
```

Comment:
Are there sometimes multiple websites split by spaces? If so - can we add `Contact` entries for each of them?